WorkloadManagementSystem/Agent/TaskQueueDirector.py

```diff
@@ -188,7 +188,9 @@ def execute( self ):
 
     self.__checkSubmitPools()
 
-    self.directorDict = getResourceDict()
+    self.directorDict = {}
+    self.directorDict['Setup'] = gConfig.getValue( '/DIRAC/Setup', 'None' )
+    self.directorDict['CPUTime'] = 9999999
```

Comment:
Well, this is what is working well in SiteDirector for years, it is like maxint in this case